src/monitoring_service/database.py

```diff
@@ -143,6 +143,27 @@ def get_channel(self, token_network_address: str, channel_id: int) -> Optional[C
     def channel_count(self) -> int:
         return self.conn.execute("SELECT count(*) FROM channel").fetchone()[0]
 
+    def get_waiting_transactions(self) -> List[str]:
```

Comment:
if this list tends to get big maybe you should consider implementing iterator (in the future).

Reply:
Good point, but this list should never get that big.